microsoft-edge/devtools-guide-chromium/rendering-tools/index.md

```diff
@@ -20,7 +20,7 @@ ms.date: 05/04/2021
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Analyze runtime performance
+# Troubleshooting common performance issues
```

Comment:
@zoherghadyali - FTS the repo, update all link text:   
![image](https://user-images.githubusercontent.com/45407486/220989066-3fee82c3-4799-446e-92b7-4b0319cb387f.png)